app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +24,9 @@
                 android:label="@string/bulk_activity_title"
                 android:parentActivityName=".ui.MainActivity"
                 android:windowSoftInputMode="adjustResize" />
+        <activity
+                android:name=".ui.QRCodeScanner"
+                android:parentActivityName=".ui.MainActivity" />
```

Comment:
Can you change the parent activity of the scanner to `.ui.BulkWriteActivity` instead?
I think it would be better for the user flow to go to that screen instead of the main screen when pressing the back button.